src/components/cc-donate.vue

```diff
@@ -0,0 +1,259 @@
+            <span
+              @click="closeModal"
+              class="close-button"
+              aria-label="Close the Donation Modal"
+              role="button"
+              >&times;</span
+            >
+          </div>
+          <div>
+            <figure
+              class="image-container"
+              style="margin: auto; text-align: center"
+            >
+              <img
+                src="https://live.staticflickr.com/65535/33949133348_e5f5d3c201_b.jpg"
+                alt="Faces of the Commons 2019"
+              />
+              <p style="font-size: 0.9rem; font-style: italic">
+                <a href="https://www.flickr.com/photos/31013861@N00/33949133348"
+                  >"Faces of the Commons 2019"</a
+                ><span>
+                  by
+                  <a href="https://www.flickr.com/photos/31013861@N00"
+                    >Sebastiaan ter Burg</a
+                  ></span
+                >
+                is licensed under
+                <a :href="donationUrl" style="margin-right: 5px">CC BY 2.0</a>
+              </p>
+            </figure>
+          </div>
+          <div class="content">
+            <p style="text-align: center">
+              Creative Commons is proud to have provided free, flexible, and
+              reusable tools powering much of the internet for nearly 20 years.
+            </p>
+          </div>
+          <div class="break-column"></div>
+          <div class="content">
+            <p style="text-align: center; font-weight: bold">
+              Please donate what you can to support the global open movement!
+            </p>
+            <p id="popup-button">
+              <a
+                href="https://www.classy.org/give/306414/#!/donation/checkout?c_src=website&c_src2=lightboxA"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Donate</a
+              >
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </transition>
+</template>
+<script>
+import { defineComponent } from "vue";
+import Cookies from "js-cookie";
+
+export default defineComponent({
+  name: "cc-explore",
+  mounted() {
+    var userHasOptedOutFromDonationAppeal = Cookies.get("optOutDonationAppeal");
+
+    if (userHasOptedOutFromDonationAppeal) {
+      this.show = false;
+      this.transition = false;
+    }
+    console.log("opted out?: ", userHasOptedOutFromDonationAppeal);
+  },
+  data() {
+    return {
+      show: true,
+      transition: true,
+      expiryPeriodInDays: 5,
+      donationUrl:
+        "https://www.classy.org/give/306414/#!/donation/checkout?c_src=website&c_src2=lightboxA",
+    };
+  },
+  methods: {
+    closeModal() {
+      this.show = false;
+      this.optOutDonationAppeal();
+    },
+    optOutDonationAppeal() {
+      Cookies.set("optOutDonationAppeal", "true", {
+        expires: this.expiryPeriodInDays,
+      });
+      Cookies.set("_ga", undefined);
```

Comment:
I'm not sure we want to clear our Google Analytics cookie when users opt-out of the donation modal. GA seems like a separate concern.